lib/archethic/contracts/interpreter/scope.ex

```diff
@@ -3,7 +3,7 @@ defmodule Archethic.Contracts.Interpreter.Scope do
   Helper functions to deal with scopes
   """
 
-  defp init(global_variables) do
+  def init(global_variables) do
```

Comment:
**IMPORTANT:**
You should not make this function public to use it in tests.
Instead you can do, as other tests, create you map and use `Process.put` to create the scope (as the init function do)